src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -564,3 +565,18 @@ def ls_dir(self, fs_id, path=None, depth=1):
         except (cephfs.PermissionError, cephfs.ObjectNotFound):  # pragma: no cover
             paths = []
         return paths
+
+
+
+@APIRouter('/cephfs/subvolume/group', Scope.CEPHFS)
+@APIDoc("Cephfs Subvolume Group Management API", "Cephfs")
+class CephFSSubvolumeGroups(RESTController):
+    def __init__(self):  # pragma: no cover
+        super().__init__()
+
+    def get(self, vol_name):
+        error_code, out, err = mgr.remote('volumes', '_cmd_fs_subvolumegroup_ls', None, {'vol_name': vol_name})
+        if error_code != 0:
+            raise RuntimeError(
+                f'Error listing subvolume groups')
+        return json.loads(out)
```

Comment:
Do we add some doc text for our swagger API? So to tell what needs to be passed to the API endpoint to get output.

Also. when we don't pass the vol_name, what will happen? Can it fetch us all the volume in that case?